radicle-cob/src/change_graph.rs

```diff
@@ -15,7 +15,10 @@ use petgraph::{
 };
 
 use crate::{
-    change, object, signatures::Signatures, Change, CollaborativeObject, ObjectId, TypeName,
+    change,
+    object::{self, ObjectIdentifier},
+    signatures::Signatures,
+    Change, CollaborativeObject, ObjectId, TypeName,
```

Comment:
Not particular to this PR, we should come to consensus on how imports are defined